src/python/CRABClient/UserUtilities.py

```diff
@@ -132,7 +132,15 @@ def getFileFromURL(url, filename = None, proxyfilename = None):
         socket = opener.open(url)
         status = socket.getcode()
         #TODO Emilis will optimize and read by chunks
```

Comment:
remove the todo and say in a comment why you are doing it in a loop (memory optimization)